src/components/Auth/ButtonSubmit/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const ButtonSubmit = ({ label }) => {
+  return (
+    <div className="button-submit">
```

Comment:
why do we need to add a class name here?
I think we only need the class name in case we have to style it with CSS

Reply:
Yes, I plan to add styles and it will be in my next PR.

Reply:
IMO we can add the class name later when we add the style so we can focus on class naming on the same PR as the style 💭

Reply:
@carryall I was wrong to think that adding class names beforehand might make review process easier. Removing all the class names will make huge changes again in lots of files. 😅 Is it okay if you reviewed the class names in the next PR?

Reply:
@rafayet-monon It's ok I'll review it on the next PR 😄